tools/vscode-azurewebpubsub/src/workflows/service/restart/RestartServiceConfirmationStep.ts

```diff
@@ -0,0 +1,37 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardPromptStep, nonNullValue, UserCancelledError } from '@microsoft/vscode-azext-utils';
+import { localize } from "../../../utils";
+import { type IPickServiceContext } from '../../common/contexts';
+
+export class RestartWebPubSubConfirmationStep extends AzureWizardPromptStep<IPickServiceContext> {
+    private serviceName: string | undefined;
+
+    public async prompt(context: IPickServiceContext): Promise<void> {
+        this.serviceName = context.serviceName;
+        const prompt: string = localize('enterToRestart', 'Enter "{0}" to restart this Web PubSub resource', context.serviceName);
```

Comment:
add note similar to portal:
![image](https://github.com/Azure/azure-webpubsub/assets/668244/720ece32-311d-4516-b540-7303d876a881)

Reply:
updated